Formula/ecr-login.rb

```diff
@@ -0,0 +1,23 @@
+class EcrLogin < Formula
```

Comment:
I'm thinking the formula should be called `docker-credential-ecr-login`, since it is the binary's name and the project name